modules/cudaoptflow/misc/python/test/test_nvidiaopticalflow.py

```diff
@@ -22,15 +22,15 @@ def test_calc(self):
         cuMat1 = cv.cuda_GpuMat(npMat1)
         cuMat2 = cv.cuda_GpuMat(npMat2)
         try:
-            nvof = cv.cuda_NvidiaOpticalFlow_1_0.create(cuMat1.shape[1], cuMat1.shape[0], 5, False, False, False, 0)
+            nvof = cv.cuda_NvidiaOpticalFlow_1_0.create((npMat1.shape[1], npMat1.shape[0]), 5, False, False, False, 0)
             flow = nvof.calc(cuMat1, cuMat2, None)
```

Comment:
This test is failing.  I think is should either be fixed by returning both the `flow` and `cost` `GpuMat`'s as in https://github.com/opencv/opencv_contrib/pull/3494#issuecomment-1556825457
```
try:
    flow, cost = nvof.calc(cuMat1, cuMat2, None)
    self.assertTrue(flow.size()[1] > 0 and flow.size()[0] > 0)
    flowUpSampled = nvof.upSampler(flow, (npMat1.shape[1], npMat1.shape[0]), nvof.getGridSize(), None)
    nvof.collectGarbage()
except cv.error as e:
    if e.code == cv.Error.StsBadFunc or e.code == cv.Error.StsBadArg or e.code == cv.Error.StsNullPtr:
        self.skipTest("Algorithm is not supported in the current environment")
self.assertTrue(flowUpSampled.size()[1] > 0 and flowUpSampled.size()[0] > 0)
```
or accessing the elements of the tuple, whichever you think is cleaner
```
try:
    nvof = cv.cuda_NvidiaOpticalFlow_1_0.create((npMat1.shape[1], npMat1.shape[0]), 5, False, False, False, 0)
    flow = nvof.calc(cuMat1, cuMat2, None)
    self.assertTrue(flow[0].size()[1] > 0 and flow[0].size()[0] > 0)
    flowUpSampled = nvof.upSampler(flow[0], (npMat1.shape[1], npMat1.shape[0]), nvof.getGridSize(), None)
    nvof.collectGarbage()
except cv.error as e:
    if e.code == cv.Error.StsBadFunc or e.code == cv.Error.StsBadArg or e.code == cv.Error.StsNullPtr:
        self.skipTest("Algorithm is not supported in the current environment")
self.assertTrue(flowUpSampled.size()[1] > 0 and flowUpSampled.size()[0] > 0)
```

Reply:
Fixed.